src/pages/JobDescribtion/JobDescriptionPage.jsx

```diff
@@ -0,0 +1,66 @@
+import { Button, Grid, Typography } from "@mui/material/";
+import { v4 as uuidv4 } from "uuid";
+import PageLayout from "../../components/PageLayout";
+import JobDescriptionItem from "./components/JobDescriptionItem";
+import mockedDescription from "./mockData";
+
+function JobDescriptionPage() {
+  return (
+    <PageLayout title="Job Description">
+      <Typography
+        variant="h2"
+        sx={{
+          width: "732px",
+          height: "59px",
+          flexShrink: "0",
+          fontSize: "30px",
+          fontFamily: "Inter",
+          fontStyle: "normal",
+          fontWeight: "400",
+          lineHeight: "normal",
+          marginTop: "100px",
+          marginLeft: "auto",
+          marginRight: "auto",
```

Comment:
I think most of it should be already in the Theme. With this approach you override that and it is hard to maintain changes in the theme with this approach

Reply:
i used the css from Figma